test/reference-types.wast

```diff
@@ -1,44 +1,44 @@
-;; reftype :: anyref | funcref | exnref | nullref
+;; reftype :: externref | funcref | exnref | nullref
 
-;; t <: anyref for all reftypes t
-;; nullref <: anyref, nullref <: funcref and nullref <: exnref
+;; t <: externref for all reftypes t
+;; nullref <: externref, nullref <: funcref and nullref <: exnref
```

Comment:
The actual subtyping relationship will have to be updated, but I think that can be a separate PR as well. Can you add a TODO here?

Reply:
Oh I yikes. Indeed. I digged a bit and it touches area of the codebase I'm not comfortable touching at the moment, e.g. fuzzer, so I'll indeed comment this as a TODO comment. Thanks!